Blog: Database improvements including MySQL 8.4

rfay opened the pull request:
## The Issue

Database improvements blog

Rendered at https://20241223-database-improvemen.ddev-com-front-end.pages.dev/blog/database-improvements/

stasadev commented on src/content/blog/database-improvements.md:
```diff
@@ -0,0 +1,50 @@
+
+Each of the following was tested with a Drupal 11 (Drupal CMS) database with 1M nodes and 1M users, a 3.7 GB compressed SQL file with no private data, available in [database-performance](https://github.com/rfay/database-performance).
+
+Using OrbStack on macOS as a Docker Provider, MySQL 8.0 import time is improved by about 25% in HEAD vs v1.24.1 ([data](https://docs.google.com/spreadsheets/d/1_4VtPTi7MVt1DdppYp8sjRaHVmA-y7vaRYMlfqtpaKY/edit?usp=sharing)):
```
A quick note about the spreadsheets: I see that all these links are separate new spreadsheets, but it could be just one document with multiple sheets. There is a button at the bottom left:

![image](https://github.com/user-attachments/assets/d540ac19-3e77-4dca-9c27-eb099acb0f73)

rfay replied:
Thanks, it's a good idea, I'll consider that next time. They were separate CSVs generated by the tools, but they could have been imported into sheets. I don't know if sheets can have a link to them, but it would have been fine anyway. Thanks for clicking through.

stasadev reviewed (changes requested):
I don't know why, but all the links to the images are broken.

rfay commented:
~~I don't understand why the images are all 404. Out of paranoia I renamed all to lower-case, but that didn't fix it. ~~

~~I changed all images to png, and that also didn't fix it.~~

All the images worked OK locally

@stasadev sorted it out. I had `/public/img` in the links instead of `/img`

stasadev reviewed (approved):
Great, thank you!

rfay commented on src/content/blog/database-improvements.md:
```diff
@@ -0,0 +1,50 @@
+---
+title: "DDEV's Database Support Gets MySQL 8.4 and Better Import Speeds"
+pubDate: 2024-12-27
```
```suggestion
pubDate: 2025-01-02
```
